requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-nose>=1.3
+nose2>=0.13.0
```

Comment:
Nose2 is a direct replacement for nose. https://docs.nose2.io/en/latest/

The update was completely seamless and didn't require any code changes. The differences between Nose and Nose2 do not currently impact us: https://docs.nose2.io/en/latest/differences.html#nose2-is-not-nose

There is an argument here for using a more widely-accepted package like `pytest`. On the one hand our tests here a pretty simple. On the other, our tests are pretty simple and could be updated fairly easily. 

I didn't see a super compelling reason to make the test framework change right now but I'm very open to discussion.

Reply:
I guess my question with this is, will we be forced to change to pytest at some point, or does it seem like nose2 will continue to be maintained? If it seems like it will be maintained, then I think we should be fine and may want a research ticket or something to determine if there is some compelling reason to make the change to pytest.